docs/internals/contributing.rst

```diff
@@ -108,6 +108,14 @@ Or, to run all the datatypes tests:
 
     $ python setup.py test -s tests.datatypes
 
+Or you can use Cricket, a GUI tool for running test suites. To start cricket in the background:
+
+.. code-block:: bash
+
+    $ pip install -r requirements/tests.txt
+    $ cricket &
```

Comment:
In other places we recommend `cricket-unittests`. Does plain `cricket` behave as expected?